ecs-agent/api/resource/resource_attachment.go

```diff
@@ -74,6 +74,16 @@ const (
 	FileSystemKey           = "fileSystem"
 )
 
+var (
+	AllowedFSTypes = map[string]bool{
```

Comment:
Why do we want to export this map?

Reply:
good point! I'll restrict this to be private instead of global